pyVor/tests/testPrimitives.py

```diff
@@ -271,11 +271,6 @@ def test_to_vector(self):
         self.assertEqual(self.b.to_vector(), Vector(1, 2))
         self.assertNotEqual(self.b.to_vector(), Vector(1, 2, 0))
 
-    def test_to_array(self):
-        """Make sure the to_array method does not exist, because it's bad."""
-        with self.assertRaises(AttributeError):
-            some_array = self.b.to_array()
-
```

Comment:
Why is this commit in here? In the context of keeping an intelligible git history, this doesn't make sense considering this code doesn't contribute to the switch to the projective plane.